identity/metadata.go

```diff
@@ -0,0 +1,64 @@
+package identity
```

Comment:
I don't think this should be a top level package. I think `activation` was OK before, but if it should be its own package how about `activation/identity`?

Reply:
identity is used by miner package as well, so I don't think it's a good idea to store it in activation package